examples/geometry/juelich_tests/test8.wkt

```diff
@@ -0,0 +1 @@
+GEOMETRYCOLLECTION (POLYGON ((10 0, 0 0, 0 4, 10 4, 10 0), (5 0.1, 6 1, 7 0.1, 5 0.1), (2.78907 3.9333, 4.48158 3.9333, 3.75451 2.90826, 2.78907 3.9333)))
```

Comment:
Please remove the small gaps between wall and obstacle. We do not support accessible areas which are too small to let agents pass.

<img width="655" alt="Screenshot 2023-04-21 at 15 35 07" src="https://user-images.githubusercontent.com/13482142/233649833-6e8e4b33-cf20-4041-9863-7b788b6faa36.png">

Reply:
done